src/components/PageBody/PageCover.tsx

```diff
@@ -69,14 +69,7 @@ export async function PageCover(props: {
                     // Maximum size of the cover
                     { width: 1248 },
                 ]}
-                className={tcls(
-                    'w-full',
-                    'object-cover',
-                    'object-center',
-                    as === 'full'
```

Comment:
@sebastiangraz I can't remember or trace back a reason for this mask. I'm going to merge this PR but let me know async if you have any concern.